src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
-export interface Figure {
+/*
+  eslint max-len: ["error", { "ignoreStrings": true }]
+*/
 
+enum FigureType {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
 }
 
-export class Triangle {
+type ColorType = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: FigureType,
+  color: ColorType,
+  getArea():number,
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape = FigureType.triangle;
+
+  constructor(
+    public color :Figure['color'],
+    public a :number,
+    public b :number,
+    public c :number,
```

Comment:
First `colon` and then `space`
```suggestion
    public color: Figure['color'],
    public a: number,
    public b: number,
    public c: number,
```